src/Fixer/Basic/BracesFixer.php

```diff
@@ -169,9 +169,17 @@ protected function createConfigurationDefinition()
             ->getOption()
         ;
 
+        $positionAfterControlStructures = new FixerOptionBuilder('position_after_control_structures', 'whether the opening brace should be placed on "next" or "same" line after control structures not covered by position_after_functions_and_oop_constructs.');
```

Comment:
In theory that's what it'll do (since I'm applying to tokens returned by `getControlTokens`),  should exclude things covered by position_after_functions_and_oop_constructs as far as I can tell?